spacy_llm/tasks/util/examples.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict, List
+
+from pydantic import BaseModel
+
+
+class SpanExample(BaseModel):
+    text: str
+    entities: Dict[str, List[str]]
+
+
+class LemmaExample(BaseModel):
+    text: str
+    lemmas: List[Dict[str, str]]
+
+
+class SpanReason(BaseModel):
```

Comment:
This is a single reponse from the LLM, right?

Reply:
Also, isn't this a duplicate of https://github.com/explosion/spacy-llm/pull/180/files#diff-86272b7a9396b646b75f0cf9f7bb4a10eaf34a0dd5cfc6d637d00be93fde050dR15?

Reply:
Oof, fixed!